src/ExpenseTrackerGrupo2/Controllers/ExpenseController.cs

```diff
@@ -0,0 +1,6 @@
+namespace ExpenseTrackerGrupo2.Controllers;
+
+public class ExpenseController
+{
+
```

Comment:
Why empty class?

Reply:
This class is from the repository set up. I just updated our folder structure so I moved this class to a new directory.